src/App.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <h1>MEMORY GAMME</h1>
+  <button @click="openFullScreen" class="fullScreenButton"></button>
+  click me
+  <section class="gameBoard">
+    <Card
+      v-for="(card, index) in cardList"
+      :key="index"
+      :value="card.value"
+      :visible="card.visible"
+      :matched="card.matched"
+      :position="card.position"
+      @select-card="flipCard"
+    />
+  </section>
+  <h2>{{ MatchStatus }}</h2>
+  <button @click="restartGame" class="ressButton">Reset Gry</button>
```

Comment:
literówka `ressButton`
```suggestion
  <button @click="restartGame" class="reset-button">Reset Gry</button>
```